### PR TITLE
gather dump before you kill the process

### PR DESCRIPTION
**Pull request by deads2k**
related to https://github.com/openshift/origin/issues/9929

Need to get an etcd dump in order to debug the problem

@stevekuznetsov ptal

### CODE REVIEW

**Comment by deads2k** [comment]
[test]

File: hack/test-cmd.sh [hunk 1]
```diff
@@ -17,19 +17,20 @@ os::util::environment::setup_time_vars
 function cleanup()
 {
     out=$?
-    pkill -P $$
```
Comment (stevekuznetsov) [inline hack/test-cmd.sh]: I'm actually amazed this wasn't mucking things up ...

**Comment by stevekuznetsov** [comment]
LGTM

**Comment by deads2k** [comment]
[merge]

**Comment by deads2k** [comment]
re[test]

**Comment by deads2k** [comment]
re[test]

**Comment by deads2k** [comment]
Yum re[test]

**Comment by deads2k** [comment]
yum re[test] re[merge]

**Pull request merged**